test/suites/dev/moonbase/test-staking/test-staking-locks8.ts

```diff
@@ -7,12 +7,15 @@ import {
   baltathar,
   generateKeyringPair,
 } from "@moonwall/util";
-import { fromBytes } from "viem";
-import { jumpRounds } from "../../../../helpers";
+import {
+  jumpRounds,
+  getDelegatorStakingFreeze,
+  getNumberOfDelegatorFreezes,
+} from "../../../../helpers";
 
 describeSuite({
-  id: "D023483",
-  title: "Staking - Locks - multiple delegations single revoke",
+  id: "D013481",
```

Comment:
This change was also caused by a conflict, we should revert it to keep the test id order.